airflow/providers/amazon/aws/hooks/s3.py

```diff
@@ -848,3 +848,74 @@ def generate_presigned_url(
         except ClientError as e:
             self.log.error(e.response["Error"]["Message"])
             return None
+
+    @provide_bucket_name
+    def get_bucket_tagging(self, bucket_name: Optional[str] = None) -> Optional[List[Dict[str, str]]]:
+        """
+        Gets a List of tags from a bucket.
+
+        :param bucket_name: The name of the bucket.
+        :type bucket_name: str
+        :return: A List containing the key/value pairs for the tags
+        :rtype: Optional[List[Dict[str, str]]]
```

Comment:
Why a list of dicts and not just one dict with the "Name" and. "Value" flattened?

Reply:
This mirrors how the API behaves.  I'm not positive why the API is returning it that way, but didn't presume to change it and possibly break someone's implementation.  It's an easy enough change if we would prefer to unwrap it.  What do you think?

Reference: https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/s3.html#S3.Client.get_bucket_tagging

Reply:
I'd prefer that we keep the standard S3 Boto API behaviour. Diverging from the API will lead to confusion, especially for folks that are moving standalone Python code over to Airflow and it's operators.

Reply:
Yep. This is what we usually prefer for the integrations/providers - to transparently move what the API we integrate with rather than fix it - this is usually more future-proof.